### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1][WIP] Table updated not conveyed.

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11257]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/platform/table/basic
If Table gets updated by any action then the same info in not conveyed to users by screen reader.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using Screen reader can check by implementing similar case. Just use table and update its column action on click of any button  and try to read this action by screen reader. Even table will reflect with updatable new value but it won't be readable by screen reader

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

**Comment by droshev** [comment]
@JyotiKumari1507 Can you provide more information, steps, scenario, usecase and JAWS history?

**Comment by JyotiKumari1507** [comment]
@droshev , If Platform table have any change in value of row, the same is reflecting in ui but screen reader is not able to read out it during testing by accessibility team. We can have more discussion in next call

**Comment by droshev** [comment]
@JyotiKumari1507 we need the input written so it will be easier to process

**Comment by JyotiKumari1507** [comment]
@droshev Please refer the attachement. We have done some update in table but after table update latest table record not read by the screen reader
<img width="278" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/157268666/e5b1cc6d-18e6-435e-be5b-13ef8b30bad2">
<img width="277" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/157268666/5ec579d6-5f0d-4b74-9ee3-45e40b3d1d2d">

**Comment by droshev** [comment]
Most of the fields are not filled.

**Comment by JyotiKumari1507** [comment]
@droshev pls let me know what input is pending from our side?

**Comment by droshev** [comment]
@JyotiKumari1507 we need stackblitz example and what the expectation is

**Comment by droshev** [comment]
@JyotiKumari1507 Can you create a stackblitz example, steps to reproduce and what the expectation is? Otherwise we will close the ticket.

**Comment by divyamalhotra488** [comment]
Hi @droshev, you can try with the example [here](https://stackblitz.com/edit/prywkj?file=package.json)

**Comment by droshev** [comment]
@divyamalhotra488 Can you share the steps and the expectation? thanks

**Comment by divyamalhotra488** [comment]
Hi @droshev, I have updated the example to reproduce the use case.
1. Now you will a 'Change Values' button at the bottom of the table in the stackblitz example.
2. Once you click it, it will update table values in UI
3. But the screen reader is not able to announce this change.

**Comment by droshev** [comment]
@divyamalhotra488 I used your stackblitz and JAWS announced the toolbar items:
<img width="664" alt="Screenshot 2024-08-05 at 11 40 17" src="https://github.com/user-attachments/assets/95332a70-6c92-40d3-b259-f3e85a4e461e">

**Comment by divyamalhotra488** [comment]
@droshev, the issue is not about the toolbar.
Screen reader should be able to read change in table cell values when 'Change Values' button is clicked.

**Comment by droshev** [comment]
@divyamalhotra488 Can you document what your expectation is?

**Comment by divyamalhotra488** [comment]
@droshev,
**Steps:**
1. Read table cell values with screen reader
2. Click on Change Values button
3. Again try to read table cell values with screen reader

**Expectation:**
The change in table cell values should be reflected with screen reader.

**Comment by droshev** [comment]
@divyamalhotra488 what i did:
1. navigate to the first cell and the screen reader read the cell data properly
2. click the button change values
3. navigate back to the table and JAWS read the new cell data.
<img width="531" alt="Screenshot 2024-08-09 at 11 30 36" src="https://github.com/user-attachments/assets/52ba7317-173a-4dc6-b7ac-497361cb59c2">
<img width="285" alt="Screenshot 2024-08-09 at 11 30 43" src="https://github.com/user-attachments/assets/b1f16547-4cf1-42cf-9d21-ea36f3264513">
I don't see any issue here of not reading the updated data
